salvo/README.md

```diff
@@ -0,0 +1,73 @@
+# salvo
+
+This is a framework that abstracts executing multiple benchmarks of the Envoy Proxy using [NightHawk](https://github.com/envoyproxy/nighthawk).
+
+## Example Control Documents
+
+The control document defines the data needed to excute a benchmark.  At the moment the dockerized scavenging benchmark is the only one supported.  To run the benchmark, create a file with the following example contents:
+
+JSON Example:
+
+```json
+{
```

Comment:
Probably out of scope, but maybe nice to track as something that would be nice-to-have in the future: These things have a tendency to get outdated; I wonder if we can generate it from the current proto definition.  E.g. in NH we have [this script](https://github.com/envoyproxy/nighthawk/blob/master/tools/update_cli_readme_documentation.sh) which generates cli options into the documentation.

Reply:
I have a `#TODO` to auto generate an example job control document.  Certainly on the radar